drivers/soundwire/cadence_master.c

```diff
@@ -224,12 +224,30 @@ static int cdns_clear_bit(struct sdw_cdns *cdns, int offset, u32 value)
 			return 0;
 
 		timeout--;
-		udelay(50);
+		usleep_range(50, 100);
```

Comment:
The patch is fine but the commit message has typos: "Parameters are provided to prevent
Slave-initiated wakes, which make be required in specific cases, and
well as controlling the restart mode." -> "Parameters are provided to prevent
Slave-initiated wakes which may be required in specific cases as well as controlling the restart mode."

Also, it is not quite clear from the commit message what "parameters" you're talking about

Reply:
reworded commit message